### PR TITLE
Update utcDCtoMODS.xsl

### PR DESCRIPTION
**Pull request by carolynfrunyon**
Correction to coll19. Sorry.

### CODE REVIEW

**Comment by markpbaggett** [comment]
@carolynfrunyon, looking at this now.

**Comment by markpbaggett** [comment]
Looks good.  No validation errors.  I added coll19 and applied the no XSL.  3093 records:

Each record has:

- an accessCondition
- a location/url
- and a titleInfo/title.

Merging.

**Comment by carolynfrunyon** [comment]
Awesome. Thanks Mark!

There are 3,284 records in our repository available for harvest. I wonder why you're only picking up 3,093?

screen shot 2017-03-15 at 8 18 33 am

**Comment by markpbaggett** [comment]
Hmmmm.  Which sets should I have? My guess is I'm missing several.

**Comment by markpbaggett** [comment]
Specifically, should I have coll1, 2, and 3? I don't right now.

**Comment by carolynfrunyon** [comment]
I think it's definitely more than 1 set, because the difference of 191 records doesn't correlate directly with any of our sets.

I just double checked, and all the the sets are listed on the XSL. Further, sets p16877coll1-p16877coll19 are available in DPLA. I did notice that there may be some missing items from various sets in DPLA. For example, for set p16877coll14, there should be 305 items, but it looks like there are only 300 of these records in DPLA.

Here's are some screen grabs of the OAI analysis.
screen shot 2017-03-15 at 8 18 33 am

screen shot 2017-03-15 at 10 00 23 am

And here's the full list of the items in DPLA:
https://dp.la/search?provider%5B%5D=University+of+Tennessee+at+Chattanooga&q=Digital+Library+of+Tennessee&sort_by=created&sort_order=desc&subject%5B%5D=Krystal--Periodicals&utf8=✓

So, in short, I don't know what's going on. Sorry, I can't shed more light on the situation. I hope I'm not opening a can of worms with this.

**Comment by carolynfrunyon** [comment]
Wrong screen grab for the first one.
screen shot 2017-03-15 at 10 01 53 am

**Comment by markpbaggett** [comment]
That's weird cause coll1, 2, and 3 aren't in repox anymore.  They haven't harvested from us since last spring.  Want me to add them back, then I can give you numbers for each set?

**Comment by carolynfrunyon** [comment]
That's so strange, because here's a link to the results from p16877coll1 in DPLA.
https://dp.la/search?utf8=✓&q=p16877coll1

**Comment by carolynfrunyon** [comment]
If it's not too much trouble, yes, please, go ahead and add them back. With the number of items/set from repox, I should be able to quickly tell you which sets are incomplete.

**Comment by markpbaggett** [comment]
I'm on the road right now, so it may be until this afternoon.   I'm starting to think I thought you wanted these removed last Spring, so I deleted them.  Maybe that's paranoia speaking.

Anyway, I'll add them back.  Rather than keep this in an unrelated p.r., let's move the discussion to an issue.  I'll open it and @ you.

**Comment by carolynfrunyon** [comment]
Sounds good, Mark. Thanks for helping me troubleshoot.
